tests/utils_tests/test_autoreload.py

```diff
@@ -102,92 +101,88 @@ def test_only_new_files(self):
         When calling a second time gen_filenames with only_new = True, only
         files from newly loaded modules should be given.
         """
-        dirname = tempfile.mkdtemp()
-        filename = os.path.join(dirname, 'test_only_new_module.py')
-        self.addCleanup(shutil.rmtree, dirname)
-        with open(filename, 'w'):
-            pass
+        with tempfile.TemporaryDirectory() as temp_dir:
```

Comment:
Maybe it's worth creating a separate class with the setUp/tearDown as you had before so all the indentation changes aren't necessary. It's just annoying for git blame to have lots of whitespace changes. I'm not sure, what do you think is nicest?